python/TestHarness/suppressions/errors.supp

```diff
@@ -120,3 +120,406 @@
+   Memcheck:User
+   fun:check_mem_is_defined_untyped
+   fun:walk_type
+   fun:walk_type_array
+   fun:check_mem_is_defined
+   fun:PMPI_Allreduce
+   fun:min<unsigned int>
+   ...
+}
+{
+   Undrs_byt_sum_ui_aui
+   Memcheck:User
+   fun:check_mem_is_defined_untyped
+   fun:walk_type
+   fun:walk_type_array
+   fun:check_mem_is_defined
+   fun:PMPI_Allreduce
+   fun:sum<unsigned int, std::allocator<unsigned int> >
+   ...
+}
+{
+   Undrs_byt_ata_lui_alui
+   Memcheck:User
+   fun:check_mem_is_defined_untyped
+   fun:walk_type
+   fun:walk_type_array
+   fun:check_mem_is_defined
+   fun:PMPI_Alltoall
+   fun:alltoall<long unsigned int, std::allocator<long unsigned int> >
+   ...
+}
+{
+   Undrs_byt_ata_pcata
+   Memcheck:User
+   fun:check_mem_is_defined_untyped
+   fun:walk_type
+   fun:walk_type_array
+   fun:check_mem_is_defined
+   fun:PMPI_Alltoall
+   fun:_ZNK7libMesh8Parallel12Communicator8alltoallImSaImEEEvRSt6vectorIT_T0_E
+   ...
+}
+{
+   Undrs_byt_max_no_t_arg
+   Memcheck:User
+   fun:check_mem_is_defined_untyped
+   fun:walk_type
+   fun:walk_type_array
+   fun:check_mem_is_defined
+   fun:PMPI_Allreduce
+   fun:max
+   ...
+}
+{
+   Undrs_byt_ata_c_ac
+   Memcheck:User
+   fun:check_mem_is_defined_untyped
+   fun:walk_type_array
+   fun:check_mem_is_defined
+   fun:PMPI_Alltoall
+   fun:alltoall<char, std::allocator<char> >
+   ...
+}
+{
+   Undrs_byt_sum_ld
+   Memcheck:User
+   fun:check_mem_is_defined_untyped
+   fun:walk_type
+   fun:walk_type_array
+   fun:check_mem_is_defined
+   fun:PMPI_Allreduce
+   fun:sum<long double>
+   ...
+}
+{
+   Undrs_byt_sum_lui
+   Memcheck:User
+   fun:check_mem_is_defined_untyped
+   fun:walk_type
+   fun:walk_type_array
+   fun:check_mem_is_defined
+   fun:PMPI_Allreduce
+   fun:sum<long unsigned int>
+   ...
+}
+{
+   Undrs_byt_max_d
+   Memcheck:User
+   fun:check_mem_is_defined_untyped
+   fun:walk_type
+   fun:walk_type_array
+   fun:check_mem_is_defined
+   fun:PMPI_Allreduce
+   fun:max<double>
+   ...
+}
+{
+   Undrs_byt_max_lui
+   Memcheck:User
+   fun:check_mem_is_defined_untyped
+   fun:walk_type
+   fun:walk_type_array
+   fun:check_mem_is_defined
+   fun:PMPI_Allreduce
+   fun:max<long unsigned int>
+   ...
+}
+{
+   Undrs_byt_sum_d_ad
+   Memcheck:User
+   fun:check_mem_is_defined_untyped
+   fun:walk_type
+   fun:walk_type_array
+   fun:check_mem_is_defined
+   fun:PMPI_Allreduce
+   fun:sum<double, std::allocator<double> >
+   ...
+}
+{
+   Unadressable_bytes_debug_0
+   Memcheck:User
+   fun:check_mem_is_defined_untyped
+   fun:walk_type
+   fun:walk_type_array
+   fun:check_mem_is_defined
+   fun:PMPI_Allreduce
+   fun:_ZNK7libMesh8Parallel12Communicator3minImEEvRT_
+   ...
+}
+{
+   Unadressable_bytes_debug_1
+   Memcheck:User
+   fun:check_mem_is_defined_untyped
+   fun:walk_type
+   fun:walk_type_array
+   fun:check_mem_is_defined
+   fun:PMPI_Allreduce
+   fun:_ZNK7libMesh8Parallel12Communicator3maxImEEvRT_
+   ...
+}
+{
+   Unadressable_bytes_debug_2
```

Comment:
My experience has been that these really specific traces tend to fail a lot as small things change on the system over time. Have you looked at putting in wildcard expressions to simplify and reduce the number of suppressions? Take a look at the existing file where we basically capture just the name of the function that we know to be bad and just wildcard most of the rest. At the very least all of these `_n` expressions can be collapsed into a single suppression.